spec/spec_helper.rb

```diff
@@ -92,3 +92,9 @@
   Kernel.srand config.seed
 =end
 end
+
+RSpec.configure do |config|
+  config.before(:suite) do
+    Rails.application.load_seed # loading seeds
```

Comment:
This is the reason your tests were failing - you had duplicates in the test database because you were running seeds AND setting up test data. You should do one or the other. At the beginning, we encourage folks to set up their own test data in the tests rather than just using seeds so that you can have more control and see what data exists in each test. You also don't need much test data in lots of the early tests. If you want to use seeds for test data, that's okay, just in that case, you shouldn't also create additional data in a before each because that will create additional data!